pipeline/inspection.py

```diff
@@ -23,6 +22,38 @@ def extract_first_number(string: str) -> Optional[str]:
             return match.group()
     return None
 
+class Organization(BaseModel):
+    """
+    Represents an organization such as a manufacturer, company, or any entity 
+    associated with a fertilizer.
+    """
+    name: Optional[str] = Field(None, description="The name of the organization.")
+    address: Optional[str] = Field(None, description="The address of the organization.")
+    website: Optional[str] = Field(None, description="The website of the organization.")
+    phone_number: Optional[str] = Field(None, description="The phone number of the organization.")
```

Comment:
Please try a prompt similar to the old one see if it picks up phone numbers in test labels 22, 24, 28 & 34.